core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts

```diff
@@ -169,6 +182,49 @@ export class TutorCardComponent {
         changes.displayedCard.currentValue)) {
       this.updateDisplayedCard();
     }
+    if (
+      this.platformFeatureService.status.EndChapterCelebration.isEnabled &&
+      this.isOnTerminalCard() &&
+      !this.animationHasPlayedOnce &&
+      this.inStoryMode
+    ) {
+      this.triggerCelebratoryAnimation();
+    }
+  }
+
+  triggerCelebratoryAnimation(): void {
+    this.checkMarkHidden = false;
+    this.checkMarkComponent.animateCheckMark();
+    this.skipClickListener = this.renderer.listen(
+      'document', 'click', () => {
+        clearTimeout(this.confettiAnimationTimeout);
+        this.checkMarkSkipped = true;
+        setTimeout(() => {
+          this.checkMarkHidden = true;
+        }, 500);
```

Comment:
Here and below, for the msec values, can we use named constants?

Reply:
Done!